Doc/library/enum.rst

```diff
@@ -405,9 +405,9 @@ Data Types
 
 .. class:: IntEnum
 
-   *IntEnum* is the same as *Enum*, but its members are also integers and can be
+   :class:`IntEnum` is the same as :class:`Enum`, but its members are also integers and can be
```

Comment:
Here (and several times below), it's again not helpful to add a link back to the current paragraph.